### PR TITLE
accessibility issue fixes

### LINKED ISSUE #12: SERVICES: SVG with image role needs alt text

### PR DESCRIPTION
**Pull request by hannakim91**
- [X] update company name svg's to have alt text displayed as header closes #12

### CODE REVIEW

File: src/testimonials/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ export const enigmaTestimonial = {
   name: 'Harley Sugarman',
   title: 'Founder, CEO at Enigma',
   imgSrc: 'harley',
+  svgAlt: 'Enigma',
```
Comment (rwwagner90) [inline src/testimonials/index.js:5]: Perhaps we should call this `companyName` or something? Then it's a little more generic for use elsewhere.

File: src/components/SingleQuote.astro [hunk 2]
```diff
@@ -21,10 +22,13 @@ const { classString, testimonial } = Astro.props as Props;
 >
   <div class="section-content bg-navy mb-16 p-4 rounded-md lg:mb-20 xl:py-20">
     <figure class="flex flex-col p-4 relative text-white xl:px-16 xl:py-0">
-      <img
-        class="h-8 max-w-full mb-3 self-start w-auto"
-        src={testimonial.svgSrc}
-      />
+      <h2>
```
Comment (rwwagner90) [inline src/components/SingleQuote.astro:25]: Is wrapping in an `h2` something that is recommended somewhere? Seems a bit odd to wrap the `img` tag.

**Comment by rwwagner90** [comment]
@hannakim91 I am not sure if you were planning to try to fix all the accessibility issues in this one PR or not, but if it's not too much trouble could we keep them small and create PRs for each open accessibility issue?

File: src/components/SingleQuote.astro (hunk 2 above)
Comment (hannakim91) [reply src/components/SingleQuote.astro:25]: I think you're right - I was thinking of the company name displayed here as a header but you probably don't need it for context for the testimonial quote

**Review by chuckcarpenter — APPROVED** [review]

**Pull request merged**